presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -252,7 +249,7 @@ public Page getOutput()
             return null;
         }
 
-        if (outputIterator == null || !outputIterator.hasNext()) {
+        if (outputIterator == null) {
```

Comment:
Ok, I remember having some weird edge cases here when testing spilling code. However I do not see why this original code should fail. Maybe it was redundant change with changes in closing operators in tests and it is not used anymore.

I agree, we can drop this and if some issue re-arise I will reimplement this in spilling PR.